tests/integration/targets/ec2_vpc_dhcp_option/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/aws
```

Comment:
```suggestion
ec2_vpc_dhcp_option_info
cloud/aws
```

I think it makes sense that we generally test _info in the same integration test set as the main module...